content/en/about/logo.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Logo"
+draft: false
+---
+
+<img src="/images/logo-big.png" class="img-fluid logo-home pr-5" style="float: left;">
+
+The [InnerSource Commons logo](/images/logo-big.png) is a swirling, white-and-black vortex with the front part in the rough shape of the letter "I" (for InnerSource).
```

Comment:
```suggestion
The [InnerSource logo](/images/logo-big.png) is a swirling, white-and-black vortex with the front part in the rough shape of the letter "I" (for InnerSource).
```

Is this the **InnerSource _Commons_** logo or just the **InnerSource** logo?

Reply:
I think it's the InnerSource logo, FWIW